plugin.video.rivedila7/addon.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.rivedila7" name="LA7-LA7d" version="5.0.0" provider-name="Luivit, Testato">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="script.module.requests" version="2.22.0"/>
+    <import addon="script.module.inputstreamhelper" version="0.5.0"/>
+    <import addon="script.module.beautifulsoup4" version="4.5.3"/>
+    <import addon="script.module.html5lib" version="0.999.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="addon.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">LA7-LA7d</summary>
+    <description lang="en_GB">Live Streaming, Replay the last week, LA7 Prime, On-Demand programs, News and Weather, Teche LA7-The Protagonists</description>
+    <summary lang="it_IT">LA7-LA7d</summary>
+    <description lang="it_IT">Diretta Live, Rivedi ultima settimana, LA7 Prime, Programmi On-Demand, Telegiornali e Meteo, Teche LA7-I Protagonisti</description>
+    <language>it</language>
+    <platform>all</platform>
+    <license>GNU General Public License v3.0</license>
```

Comment:
GPL-3.0-only or GPL-3.0-or-later (spdx identifier)

Reply:
may you made an example ? how i should change addon.xml for GPL-3.0-or-later ?

Reply:
Just use "GPL-3.0-only" (or "GPL-3.0-or-later" if you prefer) instead of the full name of the license. No need for anything else.
See spdx.org

Reply:
https://kodi.wiki/view/Addon.xml#.3Clicense.3E